src/ir/comp.rs

```diff
@@ -964,13 +965,10 @@ impl CompInfo {
         self.has_non_type_template_params
     }
 
-    /// Does this type have a virtual table?
-    pub fn has_vtable(&self, ctx: &BindgenContext) -> bool {
-        self.has_vtable ||
-        self.base_members().iter().any(|base| {
-            ctx.resolve_type(base.ty)
-                .has_vtable(ctx)
-        })
+    /// Do we see a virtual function during parsing?
+    /// Get the has_vtable boolean.
+    pub fn get_has_vtable(&self) -> bool {
+        return self.has_vtable;
```

Comment:
Let's rename these to `fn has_own_virtual_method` and `self.has_own_virtual_method` to make it more clear that we are not walking the inheritance DAG.